.github/CODEOWNERS

```diff
@@ -1,4 +1,7 @@
+# DevOps team owns the .github folder
+.github/ @aligent/devops
+bitbucket-pipelines.yml @aligent/devops
+
+# Microservice team own everything else
 * @aligent/mesh-developers
```

Comment:
There's very few people in mesh-developers, I would advocate for including microservices too (and updating the comment)